src/algebra/group_power/linear_ordered_field.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2015 Jeremy Avigad. All rights reserved.
```

Comment:
What is the best practice for these file headers? From the commit history, Yury was the author of the lemmas in this file but I also recall we should use the original copyright header.